### PR TITLE
[INLONG-2241] Add Audit Introduction

### PR DESCRIPTION
**Pull request by doleyzi**
Fixes apache/incubator-inlong#2241

### CODE REVIEW

**Review by gosonzhang — APPROVED** [review]
+1

**Comment by dockerzhang** [comment]
Please add the English document.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/design_and_concept/audit_concept.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: 审计设计
+sidebar_position: 1
+---
+
+# 总览
+
+本文尽可能全面介绍下Inlong审计的设计与实现
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/design_and_concept/audit_concept.md:8]: Add more introduction for this design.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/design_and_concept/audit_concept.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## 整体架构
+![](img/Audit_Architecture.png)
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/design_and_concept/audit_concept.md:11]: Use lowercase to name file.

**Review by dockerzhang — APPROVED** [review]

**Comment by dockerzhang** [comment]
@doleyzi be lack of English version for this new document.

File: docs/modules/audit/overview.md.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
+---
```
Comment (healchow) [inline docs/modules/audit/overview.md.md:1]: The file name contains two `.md`.

**Review by healchow — APPROVED** [review]

**Pull request merged**